xbmc/XBDateTime.cpp

```diff
@@ -1106,6 +1106,69 @@ bool CDateTime::SetFromRFC1123DateTime(const std::string &dateTime)
   return SetDateTime(year, month, day, hour, min, sec);
 }
 
+CDateTime CDateTime::FromDateString(const std::string &date)
+{
+  CDateTime dt;
+  dt.SetFromDateString(date);
```

Comment:
Can't these all not just be done by ```return dt.SetFromDateString(date);```
And would that one liner really be worth a dedicated method?